ajax/libs/html2canvas/package.json

```diff
@@ -9,20 +9,27 @@
     "screenshot",
     "html5"
   ],
-  "maintainers": [
-    {
-      "name": "Rick van Galen",
-      "email": "r.vangalen@twenteacademy.nl"
-    },
-    {
-      "name": "Jayen Ashar",
-      "email": "jayen@suburbchooser.com"
-    }
-  ],
+  "author": {
+    "name": "Niklas von Hertzen",
+    "email": "niklasvh@gmail.com",
+    "url": "http://hertzen.com"
+  },
   "repositories": [
     {
       "type": "git",
       "url": "https://github.com/niklasvh/html2canvas.git"
     }
-  ]
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/niklasvh/html2canvas.git",
+    "basePath": "dist",
+    "files": [
+      "html2canvas.js",
+      "html2canvas.min.js",
+      "html2canvas.svg.js",
+      "html2canvas.svg.minjs"
```

Comment:
My current machine can't handle the checkout, I'm just gonna close this and make a new PR. :gun: